src/__tests__/integration/upload.test.js

```diff
@@ -31,6 +26,15 @@ describe('Upload', () => {
         })
     );
 
+    it(
+        'Next step is disabled',
+        integrationTest((store, component) => {
+            expect(getDropzoneText(component)).toEqual(EMPTY_DROPZONE_TEXT);
+
+            expect(getNextStepButton(component).props().disabled).toBe(true);
```

Comment:
You can simply extend previous test with this check (as you did with `file is dropped` test). As a rule we need new test only if we introduce new actions.

Reply:
Fixed